test/JsonApiDotNetCore.MongoDb.Example.Tests/IntegrationTests/DeletingResourcesTests.cs

```diff
@@ -0,0 +1,80 @@
+using System.Net;
+using System.Threading.Tasks;
+using Bogus;
+using Example;
+using Example.Models;
+using JsonApiDotNetCore.Serialization.Objects;
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Driver;
+using Xunit;
+
+namespace JsonApiDotNetCore.MongoDb.Example.Tests.IntegrationTests
+{
+    public sealed class DeletingResourcesTests : IClassFixture<IntegrationTestContext<Startup>>, IAsyncLifetime
```

Comment:
We've recently done major work on improving the tests for create/update/delete, so we now have over 200 integration tests for that.

I think you'd benefit from taking a subset of them. A great amount can be left out, because they check on invalid request bodies. And there are lots of relationship tests.

But others, like delete-not-found, create-already-exists etc. are things you need to handle properly for MongoDb, so this repo would benefit from having those tests.